library_builders.sh

```diff
@@ -203,6 +211,43 @@ function build_blosc {
     touch blosc-stamp
 }
 
+function build_blosc2 {
+    if [ -e blosc-stamp2 ]; then return; fi
+
+    curl -sLo c-blosc2-v2.7.1.tar.gz \
+        https://github.com/Blosc/c-blosc2/archive/refs/tags/v2.7.1.tar.gz
+    file c-blosc2*.tar.gz
+    tar -xzf c-blosc2*.tar.gz
+    rm c-blosc2*.tar.gz
+
+    # @todo: proper patch for this
+    sed -E -i.bak 's|if\(\$ENV\{CMAKE_OSX_ARCHITECTURES\} STREQUAL "arm64"\)|if("$ENV{CMAKE_OSX_ARCHITECTURES}" STREQUAL "arm64")|' c-blosc2-*/CMakeLists.txt
```

Comment:
@franzpoeschel I do not see why that env var needs to be in quotes. Can you elaborate? :)
https://github.com/Blosc/c-blosc2/blob/v2.7.1/CMakeLists.txt#L243

Reply:
Oh, I see:
```
  -- Using ZSTD internal sources.
      "STREQUAL" "arm64"
```

wild!

Reply:
X-ref: https://github.com/Blosc/c-blosc2/pull/525